documentation/source/user_section/tutorials/other-features.rst

```diff
@@ -9,4 +9,13 @@ This section groups SCT features that stand on their own apart from the other se
    :maxdepth: 1
 
    other-features/spinalcord-smoothing
-   other-features/visualizing-misaligned-cords
\ No newline at end of file
+   other-features/visualizing-misaligned-cords
+
+..
+   comment:: The SCT course has a "New and Upcoming Features" section that is
+             used primarily for advertising at the end of the course. That
+             section is mostly ephemeral; whenever we start a new course, we
+             take the previous "New" features and incorporate them into the
+             main sections of the course and web tutorials. So, I don't think
+             that we need to specifically add a "New and Upcoming Features" to
+             the web tutorials.
```

Comment:
re: ed8d525e3a39a1c0f5c9274c21f590ef38d3aa8b; We don't want to have a "New and upcoming features" section in the tutorials, do we? We'd prefer to just integrate the new features directly into the web tutorials like we've always been doing?

I figure "New and upcoming features" is an SCT Course-specific construction...